### DISCUSSION TITLE
Possibility of different y-axis overlay plot

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#4332] [Community Q&A] [answered]
Suppose I want to plot four parameter in overlay plot with different ranges. Is it possible to have four y-axis in one overlay plot

**Comment by jvigliotta** [comment]
@Vaibhav2121  Could this be accomplished in an overlay plot? @shefalijoshi do you have any thoughts on this?

**Comment by shefalijoshi** [comment, marked as answer]
@Vaibhav2121 
The overlay plot supports 3 y-axis. You can have more than one parameter per y-axis.
To do this:
1. Create an overlay plot
2. Drag and drop the parameters you want to plot into this overlay plot
3. In the inspector, go to the Config tag and drag and move parameter to either y-axis 1, 2 or 3.
4. You can also show/hide different parameters using the eye icon in the overlay plot.
